x/group/client/testutil/tx.go

```diff
@@ -593,11 +576,11 @@ func (s *IntegrationTestSuite) TestTxUpdateGroupMembers() {
 	val := s.network.Validators[0]
 	clientCtx := val.ClientCtx
 
-	var commonFlags = []string{
-		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
-		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
-		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
-	}
+	weights := []string{"1", "1", "1"}
+	accounts := s.createAccounts(3)
+
+	groupID := s.createGroupWithMembers(weights, accounts)
```

Comment:
+1 for creating new groups and policies for this test to not mess up with other ones